src/suites/cts/command_buffer/render/storeop_clear.spec.ts

```diff
@@ -0,0 +1,133 @@
+export const description = `
+renderPass store op clear`;

```

Comment:
If you do the suggestion below, then the test actually becomes for both storeOps. So you could rename the file to storeop.spec.ts and update the description.